### PR TITLE
Format all config files for consistency

### PR DESCRIPTION
**Pull request by jfy133**
@bentsherman made me do it

In preparation for a build update to copy the old `max_*` parameters into the new Nextflow neative `resourceLimits` directive

### CODE REVIEW

File: conf/genouest.config [hunk 1]
```diff
@@ -17,8 +17,9 @@ process {
 
 params {
     igenomes_ignore = true
-    igenomesIgnore = true //deprecated
-    max_memory = 3000.GB
-    max_cpus = 160
-    max_time = 336.h
+    igenomesIgnore  = true
+    //deprecated
```
Comment (mirpedrol) [inline conf/genouest.config:21]:
```suggestion
    igenomesIgnore  = true //deprecated
```

File: conf/imperial.config [hunk 2]
```diff
@@ -82,21 +85,21 @@ profiles {
 }
 
 executor {
-    $pbspro {
-        queueSize = 49
+    '$pbspro' {
```
Comment (mirpedrol) [inline conf/imperial.config:88]: Is it ok to use single quotes? will the variable be replaced?

**Review by mirpedrol — APPROVED** [review]
Leaving two tiny comments, otherwise LGTM

**Comment by davidecarlson** [comment]
The seawulf config LGTM.

**Comment by wassimsalam01** [comment]
I've just opened a PR to fix mine, kindly check and approve please. Thank you

File: conf/genouest.config (hunk 1 above)
Comment (jfy133) [reply conf/genouest.config]: Will put it above

File: conf/imperial.config (hunk 2 above)
Comment (jfy133) [reply conf/imperial.config:88]: I think if it's single quotes it won't be interpreted, so I think it should be OK

File: nextflow.config [hunk 3]
```diff
@@ -1,2 +1,2 @@
 params.custom_config_base = "."
-includeConfig("nfcore_custom.config")
+includeConfig ("nfcore_custom.config")
```
Comment (mirpedrol) [inline nextflow.config:2]: I think the parenthesis are not needed here
```suggestion
includeConfig "nfcore_custom.config"
```

File: conf/imperial.config (hunk 2 above)
Comment (jfy133) [reply conf/imperial.config:88]: @bentsherman can you confirm it's OK to put in quotes?

**Comment by jfy133** [comment]
@ggrimes your profile is the one I'm least sure about the change, but we will we can use a better work around if it breaks something (`System.getenv(TMPDIR)` like other shave used)

**Review by vsmalladi — APPROVED** [review]
lgtm

**Comment by ggrimes** [comment]
> @ggrimes your profile is the one I'm least sure about the change, but we will we can use a better work around if it breaks something (`System.getenv(TMPDIR)` like other shave used)

how can I test this?

**Comment by mirpedrol** [comment]
Merging this now, please let us know if anything fails on the pipelines end

File: conf/imperial.config (hunk 2 above)
Comment (bentsherman) [reply conf/imperial.config:88]: should be fine with single quotes or no quotes

Comment (jfy133) [reply conf/imperial.config:88]: Thank you!

**Comment by jfy133** [comment]
> > @ggrimes your profile is the one I'm least sure about the change, but we will we can use a better work around if it breaks something (`System.getenv(TMPDIR)` like other shave used)
> 
> how can I test this?

Run a pipelines than use the local and pbspro executors and check the settings defined in your config are applied correctly. That said Ben said it should be fine anyway :+1:

**Pull request merged**